andela/ah-frontend-tesseract
# #160815549 - Add an image to an article

#### What does this PR do?
-  adds images to an article and profiles
#### Description of Task to be completed?
- upload an image to cloudinary
- insert an image into an article
- enable the author view an image while previewing an article
- insert an image in each article in the list of all articles
#### How should this be manually tested?
- click the camera icon on the top of the form for creating an article and then follow the appropriate steps to publish an article.
- You should be able to see an image in your article.
#### What are the relevant pivotal tracker stories?
[#160815549 ](https://www.pivotaltracker.com/story/show/160815549)

#### screenshots
#### Article view
<img width="1014" alt="screen shot 2018-10-25 at 09 27 38" src="https://user-images.githubusercontent.com/30200454/47480210-54f65c80-d838-11e8-875b-0fc9870570dc.png">

####  list of articles view
<img width="945" alt="screen shot 2018-10-25 at 09 11 59" src="https://user-images.githubusercontent.com/30200454/47479848-0a281500-d837-11e8-8a79-e6bd9bb69bdc.png">

#### Profile view
<img width="621" alt="screen shot 2018-10-25 at 09 15 06" src="https://user-images.githubusercontent.com/30200454/47479859-11e7b980-d837-11e8-921e-2ee1385ae1b7.png">
